app/results.py

```diff
@@ -65,8 +66,9 @@ def search(self, query_txt):
         # original search when they fail or return no results.
         if not results:
             if config.get('enable_fulltext_search'):
+                romanized_query = script_variant.romanize_text(query_txt)
                 results = full_text_search.search(self.repo,
```

Comment:
I think we should do romanization inside full_text_search since that's where you also do the same for indexing names.